Assets/MixedRealityToolkit/StandardAssets/Shaders/MRTK_Wireframe.shader

```diff
@@ -70,6 +70,7 @@ Shader "Mixed Reality Toolkit/Wireframe"
                 UNITY_SETUP_INSTANCE_ID(v);
                 v2g o;
                 o.viewPos = UnityObjectToClipPos(v.vertex);
+		UNITY_INITIALIZE_OUTPUT(v2g, o);
```

Comment:
@Cameron-Micka this line fixed the error, but with my lack of knowledge about shaders and this one in particular, I can't say if this shouldn't go in line 71.